Home/Home 4.1/main.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+
+enum DefaultPizzas: String {
+    case margherita = "Margherita"
+    case pepperoni = "Pepperoni"
+    case vegetarian = "Vegetarian"
+}
+
+private struct Commands {
+    static let ingredientsFile = "ingredients"
+    static let classicalOption = "classical"
+    static let customOption = "custom"
+    static let doneCommand = "done"
+    static let listCommand = "list"
+}
+
+func createDefaultPizza(_ type: DefaultPizzas, builder: PizzaBuilder) {
+    switch type {
+    case .margherita:
+        builder.addIngredientToPizza("Cheese")
+        builder.addIngredientToPizza("Tomatoes")
+    case .pepperoni:
+        builder.addIngredientToPizza("Cheese")
+        builder.addIngredientToPizza("Pepperoni")
+    case .vegetarian:
+        builder.addIngredientToPizza("Cheese")
+        builder.addIngredientToPizza("Tomatoes")
+        builder.addIngredientToPizza("Mushrooms")
+        builder.addIngredientToPizza("Peppers")
+    }
+}
+
+func main() {
+    let reader = IngredientsReader()
+    let builder = PizzaBuilder(reader: reader, filename: Commands.ingredientsFile)
+
+    print("Do you want to choose a classical pizza or compose a custom one? (classical/custom)")
+    if let choice = readLine() {
```

Comment:
those nested loops looks not very good. Consider to split them into separate functions

Reply:
Refactored the code in [00f75bc](https://github.com/YaroslavKSE/Professinal-Programming/pull/12/commits/00f75bc71bdd04cc8d3ff5c15f309f2b877968b7)